src/scripts/main.js

```diff
@@ -1,14 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notificationBlock = document.createElement('div');
+
+  notificationBlock.classList.add('notification', type);
+  notificationBlock.style.top = `${posTop}px`;
+  notificationBlock.style.right = `${posRight}px`;
+
+  notificationBlock.innerHTML
+  = `
+  <h2 class='title'>${title}</h2>
+  <p>${description}</p>
+  `;
```

Comment:
it would be better to write it in one line for better readability
```suggestion
  notificationBlock.innerHTML  = `
    <h2 class='title'>${title}</h2>
    <p>${description}</p>`;
```